.github/actions/push-rti-docker/action.yml

```
@@ -11,6 +11,10 @@ inputs:
   DOCKERHUB_TOKEN:
     description: 'The token to log in to Docker Hub'
     required: true
+  latest:
+    description: 'Also push as latest tag or not, set to true of false'
+    required: true
+    default: 'false'
```

I don't think bools need to be quoted?
```suggestion
    default: false
```

---

IntelliJ idea give an error without quotes -- I think this field has to be string